kroxylicious-operator/src/test/resources/DerivedResourcesTest/upstream-tls/in-KafkaProtocolFilter-filter-one.yaml

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+kind: KafkaProtocolFilter
```

Comment:
Does this test need a filter?

Reply:
Yes. One of the things covered by this test is that when a filter depends on the same Secret as a `KafkaService` there's a single Secret `Volume` but two `VolumeMounts`, one for each use of the certificate.